style.css

```diff
@@ -635,300 +622,231 @@ input {
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #0E1211;
-    margin-right: 5px;
-  } 
-
-  .main__product_red-circle {
-    width: 20px;
-    height: 20px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #B62E3C;
-    margin-right: 5px;
-  } 
-
-  .main__product_orange-circle {
-    width: 20px;
-    height: 20px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #FDB431;
-    margin-right: 5px;
-  } 
-
-  .main__product_copper-cirle {
-    width: 20px;
-    height: 20px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #E29D69;
-    margin-right: 5px;
-  } 
-
-  .main__product_brown-cirle {
-    width: 20px;
-    height: 20px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #762E31;
-    margin-right: 5px;
-  } 
-
-  .main__product_blue-circle {
-    width: 20px;
-    height: 20px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #41479B;
-    margin-right: 5px;
-  } 
-
-  .main__product_skyblue-circle {
-    width: 20px;
-    height: 20px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #257DA9;
-    margin-right: 5px;
-  } 
-
-  .main_mobile-back {
+  }
+
+  .main__mobile-back {
     display: block;
     position: absolute;
-    background: black; 
+    background: black;
     top: 910px;
     left: -50px;
     width: 1025px;
     height: 1600px;
-    /*margin-bottom: 90px;*/
     transform: rotate(2.6deg);
     z-index: 0;
   }
 
-  .main__video {
+  .video {
     background: none;
     margin-bottom: 125px;
   }
 
-  .main__video h3 {
+  .video__title {
     color: white;
     text-align: center;
     white-space: nowrap;
     word-break: keep-all;
   }
 
-  .main__video_img {
+  .video__img {
     margin: 0;
     display: inline-block;
     width: 100%;
   }
 
-  .main__video_about {
+  .video__about {
     position: relative;
     margin-top: 0;
     margin-right: 0;
     width: 100%;
   }
 
-  .main__video_about p {
+  .video__text {
     margin-right: 0;
     font-size: 20px;
     line-height: 25px;
     text-align: center;
   }
 
-  .main__video_about a {
+  .video__link {
     display: none;
     width: 0;
   }
-  
-  .main__stories {
+
+  .stories {
     margin-bottom: 125px;
   }
-  .main__stories-title, .main__inst-title, .main__stories_title, .main__stories_text, .main__stories_learn-more, .main__stories_date{
+
+  .stories-title
+  ,
+  .inst-title
+  ,
+  .stories__text
+  ,
+  .stories__learn-more
+  ,
+  .stories__date {
     color: white;
   }
 
-  .main__stories_item:nth-child(n+3) {
+  .stories__content:nth-child(n+3) {
     display: none;
     max-width: none;
-  } 
-  
-  .main__stories_item {
+  }
+
+  .stories__content {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".stories__content" to come before selector ".stories__content:nth-child(n+2)" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484031)